packages/web/components/UserProfile/pages.js

```diff
@@ -22,22 +16,22 @@ const getPages = (t, user, questions) => {
 			id: 1,
 			title: t('profile:userArea'),
 			pages: [
-				{ title: t('profile:myProfile'), href: '', icon: FaRegUserCircle },
-				{ title: t('profile:messages'), href: '', icon: FaEnvelope },
-				{ title: t('profile:favoriteTechnologies'), href: '/bookmarks', icon: FaHeart },
-				{ title: t('profile:userOrders'), href: '/my-orders', icon: FiShoppingBag },
+				{ title: t('profile:myProfile'), href: '', icon: User },
+				{ title: t('profile:messages'), href: '', icon: Mail },
```

Comment:
Can you change messages href so it doesn't get a bigger stroke when profile page is active?

![image](https://user-images.githubusercontent.com/13091635/102814480-79630a80-43a9-11eb-908f-8a7696ae3e91.png)